widget/ui/src/containers/SwapInput/TokenSection.styles.ts

```diff
@@ -36,6 +36,13 @@ export const TokenSectionContainer = styled('div', {
   boxSizing: 'border-box',
   alignItems: 'center',
 
+  '& ._blockchain': {
```

Comment:
In next style, we used `.token-chain-name`. Please use one convention in both cases for the naming

Reply:
I changed to blockchain-name because we have a section with this name and this section is for blockchain.